docs/adguard-browser-extension/solving-problems/update-rules.md

```diff
@@ -0,0 +1,96 @@
+---
+title: How to debug declarative rules
```

Comment:
All ad blocking rules are declarative regardless of whether it is MV3 or not.

`How to debug static rules in MV3 extension`